docs/configuration-reference.md

```diff
@@ -10,6 +10,24 @@ The Accelerated Bridge CNI configures networks through a CNI spec configuration
 * `deviceID` (string, required): A valid pci address of a SWITCHDEV NIC's VF. e.g. "0000:03:02.3".
 * `vlan` (int, optional): VLAN ID to assign for the VF. Value must be in the range 0-4094 (0 for disabled, 1-4094 for valid VLAN IDs).
 * `mac` (string, optional): MAC address to assign for the VF
+* `trunk` (array, optional): VLAN trunk configuration for the VF. 
```

Comment:
This notation seems cumbersome to me.
why not: `42,100-105`

for the user it would be easier to write this when net-attach-def is created.

Reply:
This API is copied from ovs-cni. I think it is a good idea to provide consistent APIs between CNIs from networkplumbing WG.
`42,100-105` notation also has some problems.
To support ranges (100-105) trunk field type should be a []string. 
`vlan` field is already `int`, so if a user wants to use both options right syntax will be:
```
"vlan": 10,
"trunk": ["20", "30"]
``` 
It is confusing.

Reply:
i mean:
`"trunk": "20,30,100-105,109"`

i dont think its that confusing

Reply:
also we have discussed this in the past for SR-IOV CNI 

https://github.com/k8snetworkplumbingwg/sriov-cni/issues/133

Reply:
so if this gets picked up once more for legacy SR-IOV we will be consistent with that

Reply:
Then it makes sense. I didn't know about this conversation in SR-IOV cni.
I also don't like the current syntax because it is a bit wordy. Maybe we should think about syntax change before we have CNI users

Reply:
this is not merged in sriov-cni as the issue has not matured into implementation since a plugin mechanism may be required
but the API was discussed and agreed upon there.

maybe list this as an alternative in the document for review and inputs by the community

Reply:
Ok. Sounds good